mirage/dashboard.ml

```diff
@@ -1,14 +1,110 @@
 open Tyxml
 
+let sidebar =
+  Html.(
+    div
+      ~a:[ a_class [ "w-full my-6" ] ]
+      [
+        a
+          ~a:
+            [
+              a_href "#main-content";
+              a_class
+                [
+                  "focus:absolute bg-cyan-700 hover:bg-cyan-200 text-white \
+                   px-4 py-2 rounded md:block hidden";
+                ];
```

Comment:
I'm not sure how you are supposed to achieve it, but this element is always visible while it should be hidden until focused by pressing tab.

Reply:
https://www.a11y-collective.com/blog/skip-to-main-content/#h-guide-to-incorporating-skip-to-main-content-links